src/Prism.Plugin.Popups/Extensions/PageExtensions.cs

```diff
@@ -25,14 +25,14 @@ public static class PageExtensions
         /// <returns>The displayed page.</returns>
         public static Page GetDisplayedPage(this Page page)
         {
-            while (page.IsOrDerivesFrom<MasterDetailPage>() ||
+            while (page.IsOrDerivesFrom<FlyoutPage>() ||
                   page.IsOrDerivesFrom<TabbedPage>() ||
                   page.IsOrDerivesFrom<NavigationPage>())
             {
                 switch (page)
                 {
-                    case MasterDetailPage mdp:
-                        page = mdp.Detail;
+                    case FlyoutPage flp:
+                        page = flp.Detail;
```

Comment:
I think we shouldn't have removed the `MasterDetailPage` support, but rather added `FlyoutPage` support as parallel. Otherwise, people that use `MasterDetailPage` would have a broken app.

Reply:
It appears `MasterDetailPage` derived from `FlyoutPage`, so, never mind.

Reply:
This should not break. MasterDetailPage inherits FlyoutPage